### PR TITLE
semantic

### PR DESCRIPTION
**Pull request by DzmitryKosmach**

### CODE REVIEW

File: style/main.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        font-size: 1.5em;
+}
+
+section main figure{
+	width: 128px;
+	float:left;}
+	
+.clear {
+	clear:both;
+}
```
Comment (NikitaSedyx) [inline style/main.css]: nit: empty line

File: index.html [hunk 2]
```diff
@@ -1,9 +1,76 @@
...
-        <title>Hello</title>        
-        <script src="js/main.js" type="text/javascript"></script>
-    </head>
-    <body>
-        <div>Hello</div>
-    </body>
+  <head>
+    <title>Semantic elements</title>        
+    <link href="style/main.css" rel="stylesheet" type="text/css"/>
+    <script src="js/main.js" type="text/javascript"></script>
```
Comment (NikitaSedyx) [inline index.html:5]: have you removed main.js?

File: index.html [hunk 3]
```diff
@@ -1,9 +1,76 @@
...
+          <li><a href="#">Links</a></li>
+        </ul>
+      </nav>
+      <div class="clear"></div>
+    </header>
+    <main id="main">
+      <h1>Main</h1>
+      <section>
+        <header>Header</header>
+        <main>
```
Comment (NikitaSedyx) [inline index.html]: main is unique tag in document

File: index.html [hunk 4]
```diff
@@ -1,9 +1,76 @@
...
+              survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essentially unchanged. It was
+              popularised in the 1960s with the release of Letraset sheets
+              containing Lorem Ipsum passages, and more recently with desktop
+              publishing software like Aldus PageMaker including versions of
+              Lorem Ipsum.
+            </p>
+        </main>        
+        <footer>Footer</footer>
+        <aside></aside>
```
Comment (NikitaSedyx) [inline index.html]: imo aside should be outside section, you will have a problem with css with this structure

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

**Comment by DzmitryKosmach** [comment]
https://github.com/DzmitryKosmach/DzmitryKosmach.github.io/pull/2


2017-03-27 21:18 GMT+03:00 Mikita Siadykh <notifications@github.com>:

**Review by NikitaSedyx — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**